### PR TITLE
Add modelv1 AIDO submission and final report outline

### PR DESCRIPTION
**Pull request by keyan**
- Adds a directory for the modelv1 submission, although the trained model there is pretty bad (it was trained on the smaller dataset)
- Adds a directory for the paper latex files, so far there is only an outline here, I can move this to overleaf eventually but for now wanted to keep it in source control
- Adds a script that can be used for generating training loss/accuracy graphs from tensorboard data

### CODE REVIEW

**Comment by balbok0** [comment]
Looks good to me! I will merge it and see how it works with trained model v1.
